FrameworkSystem/DB/ProxyDB.py

```diff
@@ -74,6 +84,16 @@ def __initializeDB(self):
                                      'PrimaryKey': 'Id'
                                      }
 
+    if 'ProxyDB_CleanProxies' not in tablesInDB:
+      tablesD['ProxyDB_CleanProxies'] = {'Fields': {'UserName': 'VARCHAR(64) NOT NULL',
```

Comment:
This table would be for which kind of proxies?

Reply:
This table need to store long proxies without extensions(like as dirac_group, voms)